spec/unit/recipes/configure_spec.rb

```diff
@@ -872,6 +872,10 @@
         deploy['dummy_project']['appserver']['max_pool_size'] = 10
         deploy['dummy_project']['appserver']['min_instances'] = 5
         deploy['dummy_project']['appserver']['mount_point'] = '/some/mount/point'
+        deploy['dummy_project']['appserver']['pool_idle_time'] = 300
+        deploy['dummy_project']['appserver']['max_request_queue_size'] = 100
+        deploy['dummy_project']['appserver']['error_document'] = { "503": '503.html', "504": '504.html' }
+        deploy['dummy_project']['appserver']['passenger_max_preloader_idle_time'] = 300
```

Comment:
Please, add extra tests to `it 'creates apache2 passenger config'` spec, to test if those params are actually reflected in config.

Reply:
Added tests to it 'creates apache2 passenger config'.